### ISSUE TITLE
update -templates: Initial files missing

### ISSUE THREAD

**Issue by leamas** [Rasbats/shipdriver_pi#363]
When running _update_templates_ files which are not part of the templates are not copied. When doing an actual update, this is as it should. However, it makes things harder when doing the initial run.

This includes for example the _flatpak_ directory and the _Plugin.cmake_ file. These should not be touched in a normal update, the are owned by the plugin. However, when doing the initial run on a pristine plugin it would make sense that these files were copied to allow for a reasonable starting point.

Basically, this would be about a new option -I or so which triggered copying these files and dirs.

The current docs assumes that flatpak/ is copied. However, it is not.

EDIT: An alternative solution would be to document this.

**Comment by leamas** [comment]
hm... This is really about the flatpak manifest, CMakeLists.txt/Plugin.cmake and README.md/COPYING. Updating these cannot be done in any meaningful, automated way -- it is a manual task. Which boils down to that this is a matter of documentation rather than code.

**Comment by leamas** [comment]
Submitted https://github.com/Rasbats/managed_plugins/pull/27 which should resolve this once merged

**Comment by leamas** [comment]
documentation PR (above) is merged. Closing
